project.js

```diff
@@ -0,0 +1,358 @@
+            </ul>  
+        </div>            
+
+    </div>
+    <div class="wrapper-cards">
+
+`;
+
+for(let i = 0 ; i<project_array.length; i++ ) {
+    console.log(i);
+    
+    
+    if(i==0){
+        bigString += ''; 
+        bigString += `        <div class="card-${i}">
+            <div class="up"></div>
+            <div class="down" id="orange-button-${i}"> See project</div>
+        </div>`; 
+    }
+    if(i>0){
+        bigString += `
+        <div class="card-${i}">
+            
+            <div class="${project_array[i].type_image}">  
+    
+                 <h2 class="title-post"> ${project_array[i].title_post} </h2>
+                 <h3 class="supporting-text">${project_array[i].supporting_text}</h3>
+    
+                 <div class="button-container">                         
+                 <button type="button" class="button3">${project_array[i].technologies[1]}</button>
+                 <button type="button" class="button3">${project_array[i].technologies[2]}</button>
+                 <button type="button" class="button3">${project_array[i].technologies[3]}</button>                
+                 </div>       
+                    
+            </div>  
+
+            <div class="orange-button" id="orange-button-${i}">
+                See Project
+            </div>        
+        </div>
+    `;        
+    }    
+}
+
+bigString += '</div>\n';
+
+console.log(bigString);
+
+let wrapper_works = document.querySelector('.wrapper-works');
+wrapper_works.innerHTML = bigString ;
+
+
+function create_html_popup(card)
+{       
+    let html_str=`
+    <div class="overlay">           
+    </div>
+
+    <div class="container-pop-up">
+                     
+    <div class="wrapper-0">        
+        <div class="wrapper-1">        
+            <div class="green"> 
+                <h2 class="title-multi-popup-desktop" >
+                    ${card.title_desktop}
+                </h2>
+                <h2 class="title-multi-popup-mobile" >
+                    ${card.title_mobile}
+                </h2>
+            </div>                
+            <div class="yellow"> 
+                <div class="cross-popup">     
+                <a  id="close-popup"><img src="./images/cross_2.png"></a>
+                </div> 
+            </div>            
+        </div>
+
+        <div class="wrapper-2">
+            <div class="button1">${card.technologies[1]}</div>
+            <div class="button1">${card.technologies[2]}</div>
+            <div class="button1">${card.technologies[3]}</div>                                
+        </div>  
+
+        <div class="wrapper-3">        
+            <div class="red"> 
+                <img src="${card.image}" class="image-work-popup">
+            </div>
+            <div class="wrapper-4">                
+                <div class="brown"> 
+                   <p class="works-description-popup">
```

Comment:
Kindly ensure that the popup follows the design for both the desktop and mobile versions from the chosen template in figma. :+1: 

- [ ] Kindly fix the width and height of the pop-up for both versions of mobile and desktop. It should also be in the center and responsive.
- [ ] Check the position of the `languages`, the project image and the `see source button` etc.



Your mobile pop-up            |  The figma template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-21 22-15-00](https://user-images.githubusercontent.com/65233860/197297158-a4511da8-ad6b-43e8-a440-ec910e174b17.png)  |  ![Screenshot from 2022-10-21 22-16-46](https://user-images.githubusercontent.com/65233860/197297183-5903c063-d79f-4c8c-afaa-9987004b8c69.png)


Your desktop pop-up             |  The figma template
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-21 22-15-42](https://user-images.githubusercontent.com/65233860/197297448-b6312a95-56e3-407b-8ca6-01ce346cd238.png)  |  ![Screenshot from 2022-10-21 22-14-37](https://user-images.githubusercontent.com/65233860/197297457-0554e8e2-a2e1-4018-909e-0a50bc05e6a0.png)